MathlibTest/GCongr/mod.lean

```diff
@@ -16,13 +17,16 @@ example (ha : a ≡ 4 [ZMOD 5]) (hb : b ≡ 3 [ZMOD 5]) :
   gcongr
 
 example (hb : 3 ≡ b [ZMOD 5]) : b ^ 2 ≡ 3 ^ 2 [ZMOD 5] := by gcongr
+example (hb : 3 ≡ b [ZMOD 5]) : b ^ 2 ≡ 3 ^ 2 [ZMOD 5] := by gcongr ?_ ^ 2
```

Comment:
To clarify: these tests always worked, and you merely added them? (Thanks for doing so!)